src/command_modules/azure-cli-role/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.4.0
++++++
+* `az identity` moved here from inside of the `azure-cli-vm` command_module.
+
```

Comment:
@yugangw-msft @adewaleo mind confirming this language/version choice?

Reply:
Honestly, I would argue this isn't a customer-facing change, so I would just say "minor fixes"

Reply:
LGTM. Make sure `azure-cli-vm` command module also has the similar change entry

Reply:
Looks like you two have opposite opinions on this one. I personally side with @yugangw-msft on this one, because I think that this change isn't customer facing to `azure-cli` users, but it does change the behavior of people who are tinkering with individual command modules. I've updated the `azure-cli-vm` release notes to reflect this change.

Reply:
@marstr, we do typically call out customer facing changes. But I suppose it wouldn't hurt to call this out for developers.